src/main/java/javax/ws/rs/ext/FactoryFinder.java

```diff
@@ -60,6 +62,7 @@
 final class FactoryFinder {
 
     private static final Logger LOGGER = Logger.getLogger(FactoryFinder.class.getName());
+    private static final String RESTEASY_JAXRS_API_MODULE = "org.jboss.resteasy.resteasy-jaxrs-api";
```

Comment:
actually no, for the records, as explained on chat room, this is meant to point to a private module under resteasy control